src/compile/inference/file_level_definitions.pegjs

```diff
@@ -1,39 +1,41 @@
+{
+    // Dummy uses to silence unused function TSC errors in auto-generated code
+    expected;
+    error;
+}
+
 SourceUnit =
     __ flds: (t: FileLevelDefinition __  { return t; })* {
-        // Dummy uses to silence unused function TSC errors in auto-generated code
-        expected;
-        error;
-
-        return flds as FileLevelNode<any>[];
```

Comment:
Could you restore the `as <type name>` annotations on the `return`s here and on other lines in this file?

I added those, so that if we forget a field in the grammar then the tsc compiler will complain when trying to compile the generated parser

Reply:
Will do.

Reply:
Addressed in https://github.com/ConsenSys/solc-typed-ast/pull/107/commits/c2ffabe33f1aca15df4949f269518b4b562049fa. Note that `UsingForDirective` have a typed variable, so it is protected from mentioned effect. Still, it uses a bit different notation.